src/components/button-toggle/__snapshots__/button-toggle.spec.js.snap

```diff
@@ -0,0 +1,1266 @@
+                  <StyledComponent
+                    forwardedComponent={
+                      Object {
+                        "$$typeof": Symbol(react.forward_ref),
+                        "attrs": Array [],
+                        "componentStyle": ComponentStyle {
+                          "componentId": "c4",
+                          "isStatic": true,
+                          "lastClassName": "c5",
+                          "rules": Array [
+                            "
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+",
+                          ],
+                        },
+                        "displayName": "styled.div",
+                        "foldedComponentIds": Array [],
+                        "render": [Function],
+                        "styledComponentId": "c4",
+                        "target": "div",
+                        "toString": [Function],
+                        "warnTooManyClasses": [Function],
+                        "withComponent": [Function],
+                      }
+                    }
+                    forwardedRef={null}
+                  >
+                    <div
+                      className="c4 c5"
+                    >
+                      Text
+                    </div>
+                  </StyledComponent>
+                </styled.div>
+              </label>
+            </StyledComponent>
+          </styled.label>
+        </div>
+      </StyledComponent>
+    </styled.div>
+  </ButtonToggle>
+</div>
+`;
+
+exports[`ButtonToggle Modern themes renders correctly with small theme 1`] = `
+.c4 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+  -webkit-justify-content: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+}
+
+.c5 .c3 {
+  -webkit-flex-direction: column;
+  -ms-flex-direction: column;
+  flex-direction: column;
+}
+
+.c2 {
+  display: inline-block;
+  height: 38px;
+  padding: 0 24px;
+  font-weight: 600;
+  border: 1px solid #668491;
+  font-size: 14px;
+  background-color: #FFFFFF;
+}
+
+input:checked ~ .c2 {
+  color: #FFFFFF;
+  background-color: #003F2E;
+}
+
+input:focus ~ .c2 {
+  outline: 3px solid #FFB500;
+}
+
+.c2:hover {
+  background-color: #E5F5F1;
+  border-color: #003F2E;
+}
+
+.c0 {
+  display: inline-block;
+}
+
+.c0:not(:first-of-type) {
+  margin-left: 10px;
+}
+
+.c1 {
+  position: absolute;
+  width: 0;
+  height: 0;
+  opacity: 0;
+}
+
+<div
+  className="c0"
+  size="large"
+>
+  <input
+    className="c1"
+    id="guid-12345"
+    type="radio"
+  />
+  <label
+    className="c2"
+    htmlFor="guid-12345"
+    size="large"
+  >
+    <div
+      className="c3 c4"
+    >
+      Button
+    </div>
+  </label>
+</div>
+`;
```

Comment:
another snapshot composed mostly of theme objects. Please use `TestRenderer.create` to render it properly.